docs/dictionary/command/reply.lcdoc

```diff
@@ -49,7 +49,7 @@ pieces you want to reply with.
 
 The form
 
-    reply <string> 
+    reply <string> &nbsp;
```

Comment:
@livecodesam the `&nbsp;` does not seem to make any difference. I suggest just to remove the `<>` (e.g. replace all 4 occurrences of `<string>` with `string` in this section